dist/react-dashboard.js

```diff
@@ -35443,7 +35445,9 @@ var ListWidget = React.createClass({
     getDefaultProps: function getDefaultProps() {
         return {
             pollInterval: 15000,
-            editMode: false
+            editMode: false,
+            scrollable: false,
+            maxHeight: "226px"
```

Comment:
How is the maximum height determined at the moment?  (What is the significance of 226px?)